spyder/plugins/editor.py

```diff
@@ -1002,10 +1012,12 @@ def get_plugin_actions(self):
         # ---- Edit menu/toolbar construction ----
         self.edit_menu_actions = [self.toggle_comment_action,
                                   blockcomment_action, unblockcomment_action,
-                                  self.indent_action, self.unindent_action]
+                                  self.indent_action, self.unindent_action,
+                                  self.text_uppercase_action, self.text_lowercase_action]
         self.main.edit_menu_actions += [None]+self.edit_menu_actions
         edit_toolbar_actions = [self.toggle_comment_action,
-                                self.unindent_action, self.indent_action]
+                                self.unindent_action, self.indent_action,
+                                self.text_uppercase_action, self.text_lowercase_action]

```

Comment:
These actions don't need to be added to the edit toolbar, so please remove them from here :-)